### ISSUE TITLE
Chart does not destroy properly

### ISSUE THREAD

**Issue by arthoul** [highcharts/highcharts#7021]
#### Expected behaviour
I want to be able to destroy a chart with an "areasplinerange" series and a "spline" one.

#### Actual behaviour
If I update range and then, destroy the chart, I get the following error: 
"Cannot read property 'isSVG' of undefined"

#### Live demo with steps to reproduce
click on the "1m" button and then on "destroy chart"
http://jsfiddle.net/n2dy4xk3/2/

#### Affected browser(s)
all

**Comment by KacperMadej** [comment]
@arthoul Please clear yours browser cache and try again - this error might happen when trying to use current / new Highcharts / Highstock with an exporting module in older version. I am unable to recreate the problem.

**Comment by arthoul** [comment]
Hi @KacperMadej ,
Same trouble with no cache enabled (tested on IE+Chrome on Windows 7 & Chrome on macos).
You cannot reproduce it with the jsfiddle ?

**Comment by KacperMadej** [comment]
@arthoul Tooltip had to be triggered at least once - I have recreate the problem. Thank you for your help and for reporting the problem.

**Comment by arthoul** [comment]
@KacperMadej you got it ! thank's to you

**Comment by KacperMadej** [comment]
@arthoul The problem is fixed on hc5-fixes branch, so you could use that version of code until next official release (the issue will be closed when the commit will be included in next release candidate version of Highcharts) or wait for the next Highcharts version with the fix included.

Working demo: http://jsfiddle.net/n2dy4xk3/4/

**Comment by JDoro** [comment]
Is this getting merged any time soon?

**Comment by pawelfus** [comment]
The fix is merged into master branch, that means it will be included in the next release.

**Comment by TorsteinHonsi** [comment]
It will ship within a few weeks.

Here's a drop-in fix that you can use in the meantime: 

```js
Highcharts.wrap(Highcharts.seriesTypes.arearange.prototype.pointClass.prototype, 'setState', function (proceed) {
	proceed.apply(this, Array.prototype.slice.call(arguments, 1));
    
    if (this.series.stateMarkerGraphic) {
    	this.series.lowerStateMarkerGraphic = undefined;
    }
});
```

View it live at http://jsfiddle.net/highcharts/n2dy4xk3/5/

**Comment by JDoro** [comment]
Awesome! This worked! Thanks!

**Comment by dt-karl-kuhn** [comment]
Hi, what release is this included in?

**Comment by sebastianbochan** [comment]
The issue is fixed since 6.0.0 version.

Change log: https://www.highcharts.com/documentation/changelog
